### DISCUSSION TITLE
Async call of actors does not work as discussed in the docs.

### DISCUSSION THREAD

**Discussion by brettc** [dask/distributed#7045] [Q&A] [answered]
The documentation suggests you can `await` the creation of an actor. Using the Counter class in the Actor docs, it shows:
```
async def f():
    counter = await client.submit(Counter, actor=True)
```

When I try this I get:
```
AttributeError: type object 'Counter' has no attribute '__await__'. Did you mean: '__init__'?
```

What have I misunderstood?

**Comment by brettc** [comment, marked as answer]
I'm going to answer my own question: You need to start the client with the `asynchronous=True`.

RTFM. sigh.

**Reply by mbbyn** [reply]
saved my day
